src/components/address-search/AddressSearchInput.js

```diff
@@ -9,34 +9,43 @@ import Loader from '$components/loader/Loader';
 import LoaderWrapper from '$components/loader/LoaderWrapper';
 import {stringifyQuery} from '$src/api/createUrl';
 import {KeyCodes} from '$src/enums';
-import {findFromOcdString, hasNumber} from '$util/helpers';
+import {findFromOcdString} from '$util/helpers';
 
 const SERVICE_MAP_URL = 'https://api.hel.fi/servicemap/v2';
 type Language = 'fi' | 'sv';
+const MINIMUM_SEARCH_STRING = 4
```

Comment:
Is 4 really the best? For example GitLab uses 3 as min value for searching.

Reply:
It probably doesn't matter very. I changed it to 3. The main point is to not make queries with extremely short inputs.